pipelines/logsearch.yml

```diff
@@ -99,3 +112,69 @@ jobs:
       params:
         alert_type: failed
         mode: normal
+  - name: build-logsearch-plugins-release
+    build_log_retention:
+      days: 7
+      minimum_succeeded_builds: 1
+    plan:
+      - in_parallel:
+          - get: pipeline-tasks
+          - get: logsearch-plugins-repo
+            trigger: true
+      - task: build-release
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: starkandwayne/concourse
+          inputs:
+            - name: logsearch-plugins-repo
+          outputs:
+            - name: release
+            - name: info
+          run:
+            path: /bin/bash
+            args:
+              - -c
+              - |
+                set -e
+                set -o pipefail
+
+                version=$(cat logsearch-plugins-repo/version)
+
+                printf "Building logsearch-plugins-release for version v%s\n" ${version}
+
+                pushd logsearch-plugins-repo >/dev/null
```

Comment:
Just curious why do you redirect stdout to dev null on directory changes?

Reply:
Just so there is no output in the logs